### PR TITLE
First pass at reducing number of radix colours

### PR DESCRIPTION
**Pull request by saltcod**
— excludes Radix colours we import to just the ones we need reducing colours from 700+ to ~ 400 or so.
— cleans up ui package to use the common tailwind config
— removes some config duplication — all of the different tailwind configs seemed to be duplicate, so I pointed everything at one single one

### CODE REVIEW

**Review by Ziinc — APPROVED** [review]
👍 LGTM

File: packages/config/ui.config.js [hunk 1]
```diff
@@ -6,7 +6,29 @@ const brandColors = require('./default-colors')
 
 const { default: flattenColorPalette } = require('tailwindcss/lib/util/flattenColorPalette')
 
-// console.log(Object.keys(radixUiColors))
+// exclude these colors from the included set from Radix
+const excludedRadixColors = [
+  'bronze',
```
Comment (MildTomato) [inline packages/config/ui.config.js:11]: we use bronze

File: packages/config/ui.config.js [hunk 2]
```diff
@@ -6,7 +6,29 @@ const brandColors = require('./default-colors')
...
+  'brown',
+  'crimson',
```
Comment (MildTomato) [inline packages/config/ui.config.js:13]: we use crimson in auth ui example

File: packages/config/ui.config.js [hunk 3]
```diff
@@ -6,7 +6,29 @@ const brandColors = require('./default-colors')
...
+  'cyan',
+  'gold',
```
Comment (MildTomato) [inline packages/config/ui.config.js:15]: we use gold in some LW stuff

File: packages/config/ui.config.js [hunk 4]
```diff
@@ -6,7 +6,29 @@ const brandColors = require('./default-colors')
...
+  'grass',
+  'olive',
+  'mauve',
+  'indigo',
```
Comment (MildTomato) [inline packages/config/ui.config.js:19]: we use indigo on realtime marketing page

File: packages/config/ui.config.js [hunk 5]
```diff
@@ -6,7 +6,29 @@ const brandColors = require('./default-colors')
...
+  'mint',
+  'lime',
+  'pink',
```
Comment (MildTomato) [inline packages/config/ui.config.js:22]: used in LW page

File: packages/config/ui.config.js [hunk 6]
```diff
@@ -6,7 +6,29 @@ const brandColors = require('./default-colors')
...
+  'plum',
+  'purple',
```
Comment (MildTomato) [inline packages/config/ui.config.js:24]: we use purple in LW

File: packages/config/ui.config.js [hunk 7]
```diff
@@ -6,7 +6,29 @@ const brandColors = require('./default-colors')
...
+  'sage',
+  'sand',
```
Comment (MildTomato) [inline packages/config/ui.config.js:26]: sand used once here
https://github.com/supabase/supabase/blob/master/studio/pages/join/index.tsx/#L183-L186

File: packages/config/ui.config.js [hunk 8]
```diff
@@ -6,7 +6,29 @@ const brandColors = require('./default-colors')
...
+  'sky',
+  'teal',
+  'tomato',
```
Comment (MildTomato) [inline packages/config/ui.config.js:29]: tomato used in realtime marketing page

File: packages/config/ui.config.js [hunk 9]
```diff
@@ -6,7 +6,29 @@ const brandColors = require('./default-colors')
...
+  'violet',
```
Comment (MildTomato) [inline packages/config/ui.config.js:30]: violet used once here
https://github.com/supabase/supabase/blob/master/apps/www/pages/beta.tsx/#L56

File: packages/ui/package.json [hunk 10]
```diff
@@ -22,6 +22,7 @@
     "@tailwindcss/forms": "^0.4.0",
     "@tailwindcss/typography": "^0.5.0",
     "autoprefixer": "^10.4.2",
+    "config": "*",
```
Comment (MildTomato) [inline packages/ui/package.json:25]: huh?

File: packages/ui/src/lib/theme/defaultTheme.ts [hunk 11]
```diff
@@ -239,7 +239,6 @@ export default {
       yellow: `bg-yellow-200 text-yellow-1100 border border-yellow-700`,
```
Comment (MildTomato) [inline packages/ui/src/lib/theme/defaultTheme.ts:239]: can remove any color on here in the excluded list above

File: packages/config/ui.config.js (hunk 2 above)
Comment (MildTomato) [inline packages/config/ui.config.js:13]: leaving a review comment to ask for changes. see comments above.

**Review by MildTomato — CHANGES_REQUESTED** [review]

**Review by MildTomato — APPROVED** [review]

**Pull request merged**